content/SCALE/SCALETutorials/Credentials/DirectoryServices/ConfigADSCALE.md

```diff
@@ -79,75 +119,37 @@ To check the AD join progress, open the <i class="material-icons" aria-hidden="t
 
 If you are using Windows Server with 2008 R2 or older, try creating a **Computer** entry on the Windows server Organizational Unit (OU).
 
 When creating the entry, enter the TrueNAS hostname in the name field and make sure it matches the:
 
-* *Hostname*: Go to **Network** and find **Hostname** in the **Global Configuration** window.
-* *NetBIOS alias*: Go to **Credentials > Directory Services** and click **Settings** in the **Active Directory** window. Click **Advanced Options** and find the **NetBIOS alias**.
-
-## Using Shell Commands
-
-You can go to **System Settings > Shell** and enter various commands to get more details about the AD connection and users:
-
-*AD current state*: `midclt call activedirectory.get_state`.
-
-*Connected LDAP server details*: `midclt call activedirectory.domain_info | jq`. For example:
-  ```
-  truenas# midclt call activedirectory.domain_info | jq
-  {
-    "LDAP server": "192.168.1.125",
-    "LDAP server name": "DC01.HOMEDOM.FUN",
-    "Realm": "HOMEDOM.FUN",
-    "Bind Path": "dc=HOMEDOM,dc=FUN",
-    "LDAP port": 389,
-    "Server time": 1593026080,
-    "KDC server": "192.168.1.125",
-    "Server time offset": 5,
-    "Last machine account password change": 1592423446
-  }
-  ```
-
-*View AD users*: `wbinfo -u`.
-* Enter `getent passwd DOMAIN\\<user>` to see more user details (`<user>` = desired user name).
-* If `wbinfo -u` shows more users than are available when configuring permissions and the TrueNAS cache is enabled, go to **Directory Services**, click *Settings* in the *Active Directory* window, and increase the *AD Timeout* value.
-
-*View AD groups*: `wbinfo -g`. Enter `getent group DOMAIN\\domain\ users` to see more details.
-
-*View domains*: `wbinfo -m`.
+* **Hostname**: Go to **Network** and find **Hostname** in the **Global Configuration** window.
+* **NetBIOS alias**: Go to **Credentials > Directory Services** and click **Settings** in the **Active Directory** window. Click **Advanced Options** and find the **NetBIOS alias**.
 
-*Test AD connection*: `wbinfo -t`. 
-* A successful test shows a message like `checking the trust secret for domain YOURDOMAIN via RPC calls succeeded`.
+## Disabling Active Directory
 
-*Test user connection to SMB share*: `smbclient '//0.0.0.0/smbshare -U AD.DOMAIN.COM\user`
-* `0.0.0.0` is the server address 
-* `smbshare` is the SMB share name 
-* `AD.DOMAIN.COM` is the trusted domain
-* `user` is the user account name to authenticate.
+You can disable your AD server connection without deleting your configuration or leaving the AD domain. 
+Click **Settings** to open the **Active Directory** settings screen, then select the **Enable** checkbox to clear it, and click **Save** to disable SCALE AD service. 
+This returns you to the main **Directory Services** screen where you see the two main directory services configuration options. 
 
+To configure LDAP service, click **Configure LDAP** or To turn AD service back on. 
```

Comment:
Why are we talking about LDAP here? The two are mutually exclusive.

Reply:
removed this line